examples/await.rs

```diff
@@ -1,120 +1,120 @@
-// Remove to test otherwise CI fails because of the async closures
-#![feature(async_closure)]
-#[cfg(feature = "proc_macro_example")]
-mod mac {
-    fn stack_yield_fn() {
-        #[genawaiter::stack::stack_yield_fn(u8)]
-        async fn odds() {
-            for n in (1..).step_by(2).take_while(|&n| n < 10) {
-                genawaiter::yield_! {n}
-                // this is still `()` but for completeness you can do it.
-                let _x = genawaiter::yield_!(n);
-                genawaiter::yield_!(n)
-            }
-        }
-        genawaiter::generator_mut!(gen, odds);
-        let res = gen.into_iter().collect::<Vec<_>>();
-        assert_eq!(vec![1, 1, 1, 3, 3, 3, 5, 5, 5, 7, 7, 7, 9, 9, 9], res)
-    }
+// // Remove to test otherwise CI fails because of the async closures
+// #![feature(async_closure)]
+// #[cfg(feature = "proc_macro_example")]
+// mod mac {
+//     fn stack_yield_fn() {
+//         #[genawaiter::stack::stack_yield_fn(u8)]
```

Comment:
Yeah, probably remove it, since I can't even figure out how to get it to run 🙂 I tried uncommenting everything, and I get compile errors.

Side note about dev workflow – when I made this I found running the tests to be the easiest way to work. You can just run cargo test --features proc_macro and that's about as easy as it gets. Plus, tests have the ongoing benefit of making sure the code stays working.